decentraland/unity-renderer
# Feat: add hide scene UI panel and controls

## What does this PR change?

Fix #1882
Adds a new panel under the minimap with an option to hide and show scenes UI

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/scene-ui
2. Go to wondermine or other scenes with custom UI
3. Toggle on and off the scene ui and verify the UI behavior

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
